optuna/storages/base.py

```diff
@@ -194,31 +529,79 @@ def get_best_trial(self, study_id):
 
         return copy.deepcopy(best_trial)
 
-    def get_trial_params(self, trial_id):
-        # type: (int) -> Dict[str, Any]
+    def get_trial_params(self, trial_id: int) -> Dict[str, Any]:
+        """Read parameter dictionary of a specified trial.
+
+        Args:
+            trial_id:
+                A trial ID of a trial to read parameters.
+
+        Returns:
+            A dictionary of a parameters consisting of parameter names and internal
+            representations of the parameters' values.
 
+        Raises:
+            :exc:`KeyError`:
+                If no trial with the matching `trial_id` exists.
+        """
         return self.get_trial(trial_id).params
 
-    def get_trial_user_attrs(self, trial_id):
-        # type: (int) -> Dict[str, Any]
+    def get_trial_user_attrs(self, trial_id: int) -> Dict[str, Any]:
+        """Read a user-defined attributes of a specified trial.
 
+        Args:
+            trial_id:
+                A trial ID of a trial to read user-defined attributes.
+
+        Returns:
+            A dictionary of user-defined attributes of the trial.
+
+        Raises:
+            :exc:`KeyError`:
+                If no trial with the matching `trial_id` exists.
+        """
         return self.get_trial(trial_id).user_attrs
 
-    def get_trial_system_attrs(self, trial_id):
-        # type: (int) -> Dict[str, Any]
+    def get_trial_system_attrs(self, trial_id: int) -> Dict[str, Any]:
+        """Read an optuna-internal attributes of a specified trial.
 
-        return self.get_trial(trial_id).system_attrs
+        Args:
+            trial_id:
+                A trial ID of a trial to read optuna-internal attributes.
 
-    def remove_session(self):
-        # type: () -> None
+        Returns:
+            A dictionary of optuna-internal attributes of the trial.
 
+        Raises:
+            :exc:`KeyError`:
+                If no trial with the matching `trial_id` exists.
+        """
+        return self.get_trial(trial_id).system_attrs
+
+    def remove_session(self) -> None:
+        """Clean up all connections to a database."""
         pass
 
-    def check_trial_is_updatable(self, trial_id, trial_state):
-        # type: (int, TrialState) -> None
+    def check_trial_is_updatable(self, trial_id: int, trial_state: TrialState) -> None:
+        """Check whether a `trial_state` is updatable.
+
+        Args:
+            trial_id:
+                Trial id of a trial to update.
+                Only used for an error message.
+            trial_state:
+                Trial state to check.
 
+        Raises:
+            :exc:`RuntimeError`:
+                If the trial is already finished.
+        """
         if trial_state.is_finished():
             trial = self.get_trial(trial_id)
             raise RuntimeError(
                 "Trial#{} has already finished and can not be updated.".format(trial.number)
             )
+
+    def load(self, study_id: int) -> None:
```

Comment:
Do we need to introduce this new method in this PR?
I think that the responsibility of this PR should be simply adding the doc to **the existing methods**. So, IMO, it feels better to separate this part into another PR (and it could be easier to review if the implementation and usage code of this method are also included in the PR).

Reply:
I left this method because an internal committee discussion reached an agreement that we can introduce the method here, but my opinion is neutral.
I generally agree the opinion and also don't think it's a problem to change the prior decision.
I'll remove the method later.

Reply:
> an internal committer discussion reached an agreement that we can introduce the method here.
> ...
> I'll remove the method later.

Oh, sorry for my comment disregarding the previous decision, but I would appreciate it if you could do it.

Reply:
Removed the method in https://github.com/optuna/optuna/pull/1175/commits/cfd6f2128ced8a5644b66ce4040133b0e460b687.

Reply:
Thanks!